src/com/facebook/buck/rules/KnownBuildRuleTypes.java

```diff
@@ -525,7 +526,7 @@ static Builder createBuilder(
 
     KotlinBuckConfig kotlinBuckConfig = new KotlinBuckConfig(config);
 
-    ScalaBuckConfig scalaConfig = new ScalaBuckConfig(config);
+    final ScalaBuckConfig scalaConfig = new ScalaBuckConfig(config);
```

Comment:
This doesn't need to be final, I assume it's a leftover from some earlier version of the code.